main.js

```diff
@@ -201,12 +201,13 @@
                             cropToSingleLayer = typeof layerSpec == "number",
                             svgWriterErrors = [];
                         generatorPlus.patchGenerator(doc, generator, compId, cropToSingleLayer, constrainToDocBounds,
-                                                    layerSpec, layerScale, svgWriterErrors).then(function () {
+                                                    layerSpec, layerScale, svgWriterErrors, params.useFlite).then(function () {
                             if (layerSpec === "all") {
                                 layerSpec = null;
                             }
-                            logger && logger.info("SVGOMG Document: " + JSON.stringify(doc));
-                            logger && logger.info("SVGOMG Params: " + JSON.stringify(params));
+
+                            //logger && logger.debug("SVGOMG Document: " + JSON.stringify(doc));
```

Comment:
Is this now commented out?

Reply:
yes.  i think it helps to have it there (commented out) as a guide for future troubleshooting.  But it is too verbose to leave even in the "debug" logs, IMO.  I'll remove it though since this is not a very good argument.